doc/bucket-manifest.md

```diff
@@ -0,0 +1,44 @@
+# TL;DR
+
+A tool to generate an object manifest of a s3 bucket. Users can run multiple jobs simultaneously. Each job will be run on a seperated infrastructure that makes it easier to manage.
+
+The tool will spin up a compute enviroiment, a job queue, a job definition and a SQS. AWS Batch will manage the infrastructure, scaling up or down based on the number of jobs in queue. It also automatically bid on spot instances for you. The SQS is where the ouput computations are stored.
+
+The tool can be used to any kind of task that requires batch operations. Users who use AWS batch need to write a job/service to submit jobs to the queue and to consume the SQS. The repos (https://github.com/uc-cdis/aws-batch-jobs) is where all the k8s jobs consuming SQS are stored.
+
+
+## Use
+
+### create
+
+Launch a AWS batch operation job to generate a manfest of the source bucket with provided subnets. The result will be put on the manifest bucket.
+
+```bash
+  gen3 bucket-manifest create <source bucket> <subnets> <manifest bucket>
+```
```

Comment:
This is very cool, Giang!
Where are the subnets supposed to come from?  Is it expected we'll run these jobs in the same VPC as the k8s cluster?  If not - how do we spin up a new VPC with subnets and S3 endpoints and no external route?

Reply:
The subnets are from the same VPC which can be refered from `subnet_id`. It is supposed that the job would run in existed VPC and existed subnets. However, I have just made some changes in terraform code to create new VPC, routing table, subnets in 9807ce1e8b5abcb5c945294b993dc2a4db2c88cf. So the command is updated to `gen3 bucket-manifest create <source bucket> <manifest bucket>`